### ISSUE TITLE
feat: add support for disabled state in Accordion Items

### ISSUE THREAD

**Issue by brianchristopherbrady** [microsoft/fast#6568]
# 🙋 Feature Request

We need to support disabled accordion items in our design system and the keyboarding interaction model doesn't provide support for that.

## 🤔 Expected Behavior

Ability for dev to pass a disabled attribute through the Accordion Item that set's an aria-disabled state on the Accordion Item and a disabled attribute on the button.

## 😯 Current Behavior

The current behavior does not support a disabled state.

## 💁 Possible Solution

```html
<accordion-item disabled>
   <span slot="heading">Accordion Header 1</span>
   <p>Accordion Panel 1</p>
</accordion-item>
```
```ts
export function accordionItemTemplate<T extends FASTAccordionItem>(
    options: AccordionItemOptions = {}
): ElementViewTemplate<T> {
    return html<T>`
        <div
            class="heading"
            part="heading"
            role="heading"
            aria-level="${x => x.headinglevel}"
            aria-disabled="${x => x.disabled}"
        >
            <button
                class="button"
                part="button"
                ${ref("expandbutton")}
                aria-expanded="${x => x.expanded}"
                aria-controls="${x => x.id}-panel"
                id="${x => x.id}"
                @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
                disabled="${x => x.disabled}"

            >
                <span class="heading-content" part="heading-content">
                    <slot name="heading"></slot>
                </span>
            </button>
            ${startSlotTemplate(options)}
            ${endSlotTemplate(options)}
            <span class="icon" part="icon" aria-hidden="true">
                <slot name="expanded-icon">
                    ${options.expandedIcon ?? ""}
                </slot>
                <slot name="collapsed-icon">
                    ${options.collapsedIcon ?? ""}
                </slot>
            <span>
        </div>
        <div
            class="region"
            part="region"
            id="${x => x.id}-panel"
            role="region"
            aria-labelledby="${x => x.id}"
        >
            <slot></slot>
        </div>
`;
}
```

## 🔦 Context

We need to support disabled accordion items in our design system. We considered extending the Accordion Item class to accept a disabled attribute and then applying CSS to create a "disabled" state. However  based on the the keyboarding interaction model this approach does not meet the requirements.

## 💻 Examples
![image](https://user-images.githubusercontent.com/11062709/207996036-2980b2fe-9a1a-4fd1-a92f-d6cc94b2b1ba.png)

**Comment by brianchristopherbrady** [comment]
@chrisdholt happy to contribute :)

**Comment by chrisdholt** [comment]
closed with #6587
